Makefile

```diff
@@ -17,7 +17,7 @@ endif
 
 PKG=github.com/sigstore/cosign/cmd/cosign/cli
 
-LDFLAGS="-X $(PKG).gitVersion=$(GIT_VERSION) -X $(PKG).gitCommit=$(GIT_HASH) -X $(PKG).gitTreeState=$(GIT_TREESTATE)"
+LDFLAGS="-X $(PKG).gitVersion=$(GIT_VERSION) -X $(PKG).gitCommit=$(GIT_HASH) -X $(PKG).gitTreeState=$(GIT_TREESTATE) -X $(PKG).buildDate=$(BUILDDATE)"
```

Comment:
This looks great! I guess I'm still unsure on whether or not we should include the build date though, for reproducible builds. Any opinions here? I suppose people could always just set the same date...

Reply:
Use `SOURCE_DATE_EPOCH`?

https://reproducible-builds.org/docs/source-date-epoch/

Reply:
Ah nice! this is perfect. Thanks @jonjohnsonjr

Reply:
learn new thing today, thanks for that @jonjohnsonjr

Reply:
Let me know how you want to proceed